packages/arb-token-bridge-ui/src/components/common/NetworkSelectionContainer.tsx

```diff
@@ -236,14 +250,16 @@ function NetworksPanel({
     }
 
     const coreNetworks = chainIds.filter(
-      chainId => !isNetwork(chainId).isOrbitChain
+      chainId => isNetwork(chainId).isCoreChain
     )
+    const otherNetworks = chainIds.filter(chainId => isNetwork(chainId).isBase)
```

Comment:
if we ever add other chains to this group this check would fail, so can we do
```suggestion
    const otherNetworks = chainIds.filter(chainId => !isNetwork(chainId).isCoreChain && !isNetwork(chainId).isOrbitChain)
```